src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -275,17 +275,20 @@ export const FilesTableRow = ({
                           row
                           sx={{ flexWrap: 'nowrap' }}
                           onChange={(event) => {
-                            const fileVersion = event.target.value as FileVersion;
-                            setFieldValue(field.name, fileVersion);
+                            const newFileVersion = event.target.value as FileVersion;
+                            const previousFileVersion = field.value as FileVersion;
+                            setFieldValue(field.name, newFileVersion);
 
                             if (isRrsApplicableCategory) {
-                              if (fileVersion === FileVersion.Published) {
+                              if (newFileVersion === FileVersion.Published) {
                                 const nullRrsValue: FileRrs = {
                                   type: 'NullRightsRetentionStrategy',
                                   configuredType: rrsStrategy,
                                 };
                                 setFieldValue(rrsFieldName, nullRrsValue);
-                                setFieldValue(licenseFieldName, null);
+                                if (!!previousFileVersion) {
```

Comment:
Ved å være eksplisitt vil vi kanskje slippe at den nullstilles om denne måtte kalles med samme verdi som før? Usikker på om det kan skje da, for det burde vel ikke trigge en onChange 🤔 
```suggestion
                                if (previousFileVersion === FileVersion.Accepted) {
```